CONTRIBUTING.md

```diff
@@ -206,15 +206,15 @@ You must have the following installed on your system to contribute locally:
 
 #### Debian/Ubuntu
 
-`sudo apt install g++ make` meets the additional requirements to run `node-gyp` in the context of building Cypress from source.
+`sudo apt install build-essential` meets the additional requirements to run `node-gyp` in the context of building Cypress from source.
```

Comment:
@AtofStryker 

You were too fast with this PR for me to give feedback in time!

In my tests on Ubuntu and Debian, `g++ make` were sufficient for the rebuilds using `node-gyp`

`build-essential` is a superset of `g++ make`, so using this will also work.

It installs the following packages additional to `g++ make`, 
```text
  bzip2 dpkg-dev fakeroot libalgorithm-diff-perl libalgorithm-diff-xs-perl libalgorithm-merge-perl libdpkg-perl libfakeroot libfile-fcntllock-perl
  lto-disabled-list
```